controller/api/polygon.js

```diff
@@ -208,6 +210,9 @@ class PolyCtrl {
 
           const burnABI = burnNFTFunction.encodeABI();
 
+          // Need to set timeout to handle a big amount of burn calls to blockchain
+          await sleep(TRANSACTION_DELAY);
```

Comment:
I think we should retry some amount of times when tx fails and sleep before each retry.
if it retry even when it fails after timeout in current implementation, we could leave it.

Reply:
There is a retry mechanism on a transaction execution method.